tests/test_gateway.py

```diff
@@ -38,7 +38,11 @@ def generate_kernelspec(name):
             "metadata": {},
         }
     }
-    kernelspec_stanza = {"name": name, "spec": spec_stanza, "resources": {}}
+    kernelspec_stanza = {
+        "name": name,
+        "spec": spec_stanza,
+        "resources": {"logo-64x64": f"f/kernelspecs/{name}/logo-64x64.png"},
```

Comment:
It would be nice if we could validate that these are updated to include the server's `base-url`.  I believe the tests run with a `base-url` configured.  I can maybe help with this tomorrow.  We could add it later as well unless we fail the codecov here.

Reply:
I added one test case to validate this